### DISCUSSION TITLE
Amount en cardForm obligatorio

### DISCUSSION THREAD

**Discussion by eedv** [mercadopago/sdk-js#39] [Q&A] [answered]
Hola, en nuestra app hacemos recurrencia y permitimos al cliente agregar medios de pago antes hacer el cobro efectivo, pero al crear un cardForm es obligatorio poner un valor en `amount`. 
Mi duda es si es realmente necesario y qué pasaría si pusiera cualquier valor y después, en el server, cobrara por otro valor.
En el ejemplo que tienen (`card-payment-sample`) lo hice asi y cobró sin problemas pero temo que la tasa de aprobación baje por eso. 
Gracias.

**Comment by hdlopez** [comment, marked as answer]
Hola @eedv ! El amount en general se utiliza para los cálculos de cuotas en el formulario. Si tu formulario no tiene cuotas, podrías poner un monto "dummy" y utilizarlo sin problemas.

Nos llevamos el feedback para analizar el caso si tiene sentido que sea required para todos los casos.

Muchas gracias

Horacio

**Reply by eedv** [reply]
Gracias por la respuesta.

**Comment by muniodiego** [comment]
Tengo el mismo inconveniente, utilizar un importe dummy es confuso para quien realiza el pago, me parece importante que amount no sea un campo obligatorio (como sucedía en v1 de sdk-js).

**Reply by lucmantovani** [reply]
Hola, @muniodiego!

Entiendo la inconveniencia de enviar un `amount` dummy, pero por hora es el comportamiento default de Card Form

Quizás una alternativa sería hacer la [integración con los Core Methods](https://www.mercadopago.com.ar/developers/es/docs/checkout-api/integration-configuration/card/integrate-via-core-methods), que permite un mayor control sobre el flujo de pago y no necesita utilizar un `amount` dummy

**Comment by SLW108** [comment]
Buenas! como estan? 

Estaria teniendo un error relacionado al campo amount. 
Cuando quito el dummy por defecto que trae la doc y le paso un valor string a traves de una variable (para que calcule las cuotas con el valor "real" de la compra, el form me devuelve este error: {code: '000', message: 'invalid parameter amount'}. 
Por que podria ser? 
Desde ya, gracias.

**Reply by lucmantovani** [reply]
Hola @SLW108!
Mismo usando un valor dummy, el valor debe ser mayor que 0.

**Reply by SLW108** [reply]
Creo estar mandando un valor mayor a 0. 

Si ese codigo de error refiere a lo que me decis, debo tener algun problema en el viaje del valor hasta el formulario. 

Gracias!
